examples/Notebooks/Performance/DP_WSCC9bus_SGReducedOrderVBR_validation.ipynb

```diff
@@ -0,0 +1,228 @@
+{
+ "cells": [
+  {
```

Comment:
We should add validation of DenseLU against SparseLU to both validation notebooks.

Reply:
done